ansible/roles/hoover/tasks/main.yml

```diff
@@ -117,7 +117,8 @@
 - name: Download Apache Tika
   get_url:
     dest: /opt/hoover/tika-server.jar
-    url: http://apache.javapipe.com/tika/tika-server-1.17.jar
+    url: https://jenkins.liquiddemo.org/__images__/base/tika-server-1.17.jar
```

Comment:
This isn't a base image, so at least put it in some other folder, instead of `base`, like `lib`